native/src/components/NavigationTile.tsx

```diff
@@ -92,7 +92,7 @@ class NavigationTile extends React.Component<PropsType> {
   render(): ReactNode {
     const { tile, theme, width } = this.props
     return (
-      <TileTouchable theme={theme} onPress={tile.onTilePress} width={width}>
+      <TileTouchable theme={theme} onPress={tile.onTilePress ?? undefined} width={width}>
```

Comment:
This and other similar changes appear because most of the properties in React are either set or `undefined`, but `null` is not a valid value. If you have a better solution feel free to drop a comment.